cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -844,8 +844,12 @@ impl MachInst for Inst {
     }
 
     fn worst_case_size() -> CodeOffset {
-        // calculate by test function riscv64_worst_case_instruction_size()
-        124
+        // Currently our largest possible instruction is return_call_indirect.
+        // Depending on the number of clobbers, it can generate up to 172 bytes.
+        //
+        // We also have a test case that tests some other instructions which have
+        // the potential to generate a lot of bytes
+        172
```

Comment:
I agree it's theoretically hard to enumerate all the different possible `Inst`s` and programmatically test the worst case -- but could we add a test at least that manually constructs what we think is the worst-case `Inst` (`return_call_indirect` with all clobbers set, etc) and asserts its size is 172? Size tests like this for large pseudoinsts plus a social norm of adding them for new large pseudoinsts will at least give partial coverage for this issue I think...